doc/changes/changes_5.0.0.md

```diff
@@ -33,6 +34,7 @@ This release uses version 0.14.0 of the container tool.
  - #650: Changed nightly cron and avoid running nightly builds during the weekend
  - #648: Moved swig-2.0.4.tar.gz to AWS
  - #572: Refactored package import tests
+ - #590: Added minimal SLC with Conda support
```

Comment:
This is feature https://github.com/exasol/script-languages-release/issues/590 and we should rename it to 

> Added flavor template-Exasol-all-python-3.8-conda